test/quantization/pt2e/test_quantize_pt2e_qat.py

```diff
@@ -304,34 +303,70 @@ def _verify_symmetric_xnnpack_qat_graph_helper(
         self.assertEqual(eps, 1e-5)
 
 
-@skipIfNoQNNPACK
-class TestQuantizePT2EQAT(PT2EQATTestCase):
-    def test_qat_conv_no_bias(self):
-        class M(torch.nn.Module):
-            def __init__(self, has_relu: bool):
-                super().__init__()
-                self.conv = torch.nn.Conv2d(3, 3, 3, bias=False)
-                self.relu = torch.nn.ReLU() if has_relu else torch.nn.Identity()
+class _BaseTestQuantizePT2EQAT(PT2EQATTestCase):
+    """
+    Base TestCase to be used for all conv-bn[-relu] fusion patterns.
+    """
 
-            def forward(self, x):
-                x = self.conv(x)
+    class _TestModel(torch.nn.Module):
```

Comment:
how about putting these in `PT2EQATTestCase`?

Reply:
sure